tools/ui-components/src/form-group/types.ts

```diff
@@ -0,0 +1,8 @@
+import React, { ReactNode } from 'react';
+
+export interface FormGroupProps extends React.HTMLAttributes<HTMLElement> {
+  children?: ReactNode;
+  className?: string;
+  controlId?: string;
+  validationState?: ['success', 'warning', 'error', null];
+}
```

Comment:
I removed `bsClass` and `bsSize` from it, because we aren't using them.

Reply:
That is a good call.